### PR TITLE
Extract snapshot consensus

### PR DESCRIPTION
**Pull request by kpudlik**

### CODE REVIEW

**Review by marcinwadon — APPROVED** [review]
lgtm but let's wait for @TheMMaciek approval

**Comment by marcinwadon** [comment]
ps. label missing

**Comment by marcinwadon** [comment]
@kpudlik I agreed with @TheMMaciek that he'll do a post-review as it's blocking me right now

File: modules/sdk/src/main/scala/org/tessellation/sdk/snapshot/package.scala
Comment (TheMMaciek) [inline modules/sdk/src/main/scala/org/tessellation/sdk/snapshot/package.scala:10]: You don't need this type to be parameterized by `B <: Block[_]` in order to not operate on generic `Block`?

**Pull request merged**
